docs/deployment/README.md

```diff
@@ -352,15 +241,15 @@ List of EarthData users you wish to have access to your dashboard application.
     - username: <user2>
 ```
 
-##### Configure EarthData application
+### Configure EarthData application
 
 The Cumulus stack is expected to authenticate with [Earthdata Login](https://urs.earthdata.nasa.gov/documentation). You must create and register a new application. Use the [User Acceptance Tools (UAT) site](https://uat.urs.earthdata.nasa.gov) unless you changed `urs_url` above. Follow the directions on [how to register an application.](https://wiki.earthdata.nasa.gov/display/EL/How+To+Register+An+Application).  Use any url for the `Redirect URL`, it will be deleted in a later step. Also note the password in step 3 and client ID in step 4 use these to replace `clientid` and `clientpassword` in the `.env` file in the next step.
 
-##### Set up an environment file:
+### Set up an environment file
```

Comment:
We should probably add something in here about setting the `TOKEN_SECRET` value? I can do that in a separate PR if you want

Reply:
I added something but if you want more I'd be happy to do that here

Reply:
Added a comment below